privacy-security-explainer.md

```diff
@@ -57,9 +60,219 @@ Rounding, quantization, and fuzzing are three categories of mitigations that mod
+
+Developers indicate the desire for `XRViewerPose` data by calling `XRFrame.getViewerPose()`.
+
+```js
+function onSessionRafCallback(XRFrame frame) {
+    let viewerPose = frame.getViewerPose(xrReferenceSpace);
+}
+```
+
+In addition to meeting the [`XRPose`](#xrpose) requirements, every call to `XRFrame.getViewerPose()` which will return more than one `XRView` must additionally ensure that:
+* User intention is well understood, either via [explicit consent](#explicit-consent) or [implied consent](#implied-consent)
+* If `XRView` data is affected by settings that may vary from device to device, such as static interpupillary distance, variations in screen geometry, or user-configured interpupillary distance, then the XRView data must be anonymized to prevent fingerprinting. Specific approaches to this are at the discretion of the user agent.
+* If `XRView` data is affected by a user-configured interpupillary distance, then it is strongly recommended that the UA required explicit consent during the creation of the `XRReferenceSpace` passed into `XRFrame.getViewerPose()`.
+
+## Reference spaces
+### Unbounded reference spaces
+Unbounded reference spaces reveal the largest amount of spatial data and may result in user profiling and fingerprinting. For example, this data may enable determining user’s specific geographic location or to perform gait analysis.
+
+Developers indicate the desire for unbounded viewer tracking at the time of session creation by adding `unbounded` to either `XRSessionInit.requiredFeatures` or `XRSessionInit.optionalFeatures`. 
+
+```js
+function onARClick() {
+  xr.requestSession('immersive-ar', { requiredFeatures: ['unbounded'] } )
+  .then(onARSessionCreated);
+}
+```
+
+In response the UA must ensure that:
+* The document is allowed to use all the policy-controlled features associated with the sensor types used to track the native origin of an unbounded reference space
+* User intention is well understood, either via [explicit consent](#explicit-consent) or [implied consent](#implied-consent)
+* The XR device is capable of unbounded tracking
+
+If these requirements are not met and `unbounded` is listed in `XRSessionInit.requiredFeatures` then the promise returned from `requestSession()` must be rejected. Otherwise, the promise may be fulfilled but future calls to `XRSession.requestReferenceSpace()` must fail when passed `unbounded`.
+
+Once a session is created, developers may attempt to create a bounded reference space by passing `unbounded` into `XRSession.requestReferenceSpace()`.
```

Comment:
Is this supposed to be 'create an unbounded reference space' or am I missing something?